src/Analyzers/Core/CodeFixes/UseInterpolatedString/AbstractUseInterpolatedStringCodeFixProvider.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.UseInterpolatedString;
+
+internal abstract partial class AbstractUseInterpolatedStringCodeFixProvider : SyntaxEditorBasedCodeFixProvider
```

Comment:
This doesn't need to be `partial`, right?
```suggestion
internal abstract class AbstractUseInterpolatedStringCodeFixProvider : SyntaxEditorBasedCodeFixProvider
```